tests/load-tests/cluster_read_config.yaml

```diff
@@ -26,10 +26,6 @@
   monitoring_query: sum(rate(token_pool_gauge{rateLimited="secondary"}[5m]))
   monitoring_step: 15
 
-- name: measurements.cluster_nodes_worker_count
-  monitoring_query: count(kube_node_role{role="worker"})
-  monitoring_step: 15
-
```

Comment:
@naftalysh Why do you remove this metric?

Reply:
@pmacik
That was the suggestion from Jan as our Prometheus SA doesn't have authorization for that action

Reply:
Hello. I do not think I have suggested that. I do not think this query is failing, or is it?